source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -1162,8 +1162,14 @@ STATIC_ASSERT( pdMS_TO_TICKS( ipconfigPHY_LS_LOW_CHECK_TIME_MS ) <= portMAX_DELA
     #error ipconfigIP_TASK_PRIORITY must be at least 0
 #endif
 
-#if ( ipconfigIP_TASK_PRIORITY > ( configMAX_PRIORITIES - 1 ) )
-    #error ipconfigIP_TASK_PRIORITY must be at most configMAX_PRIORITIES - 1
+#if ( configENABLE_MPU != 0 )
```

Comment:
Since portPRIVILEGE_BIT is always defined, can we condense this like the following -
```c
#if ( ( ipconfigIP_TASK_PRIORITY & ~portPRIVILEGE_BIT ) > ( configMAX_PRIORITIES - 1 ) )
```